chirp/drivers/kenwood_live.py

```diff
@@ -198,39 +246,72 @@ def __init__(self, *args, **kwargs):
 
             command(self.pipe, "AI", "0")
 
-    def _cmd_get_memory(self, number):
-        return "MR", "%i,0,%03i" % (self._vfo, number)
+    def _cmd_get_memory(self, number, cmd="MR"):
+        if isinstance(number, str):
+            return cmd, "%i,0,%s" % (self._vfo, number)
```

Comment:
So these look like `U1` or something is that right? I'm not sure I like making these take a string or an int, but I acknowledge it looks like the easiest path.

Reply:
Yeah, "L0", "U0" matches what's passed in that field, and what is displayed, so I did it the hackish way.